dallinger/experiment_server/experiment_server.py

```diff
@@ -27,6 +27,7 @@
 from dallinger import db, experiment, models, recruiters
```

Comment:
I tried testing this by running `psynet debug local` in PsyNet's `experiments/hello_world` demo. 

First I tried setting `recruiter = devprolific`. I saw strange inconsistent console messages here: sometimes it would say `Initializing HotAirRecruiter`, and sometimes it would say `Initializing ProlificRecruiter`; in other words, different recruiters seemed to be used in different places. I did not see any log messages about simulated Prolific API calls though.

Then I tried additionally setting `debug_recruiter = devprolific`. I got an error message for that:

```
{"status": "error", "message": "Failed to open recruitment, check experiment server log for details: Can't run a Prolific Study from localhost"}
```

@fmhoeger do you get the same?

Reply:
Sorry, just remembered I need to use the PsyNet branch you mentioned... will report back...

Reply:
I tried with the updated PsyNet branch and got the same issue, unfortunately.

Reply:
I see. Could you try specifying `DevProlificRecruiter` instead in the config. I will try getting `devprolific` to work in the meantime.

Reply:
Sorry, I addressed your two comments above and you should now be able to debug locally with these settings:
```
recruiter = generic
debug_recruiter = devprolific
```

Reply:
Excellent, that worked, thank you!